tests/android/tests/test_android_discovery.py

```diff
@@ -0,0 +1,145 @@
+    """
+
+    def test_start_discovery_smoke(self, set_capabilities, setup_logging):
+        """
+        Scenarios:
+            Verify discovery screen is loaded successfully
+            Verify that the screen title is correct
+            Verify that the explore courses button is displayed
+            Verify that the explore courses button text is correct
+            Verify that the back button is displayed
+            Verify that the back button text is correct
+            Verify that the discovery screen message is correct
+        """
+
+        setup_logging.info(f'Starting {TestAndroidDiscovery.__name__} Test Case')
+        android_landing = AndroidLanding(set_capabilities, setup_logging)
+        global_contents = Globals(setup_logging)
+
+        assert android_landing.get_screen_title().text == values.LANDING_MESSAGE_IOS
+        assert android_landing.get_explore_courses().text == values.LANDING_EXLPORE_COURSES
+        android_landing.get_explore_courses().click()
+
+        back_button = global_contents.get_back_button(set_capabilities)
+        assert back_button.get_attribute('displayed') == values.TRUE_LOWERCASE
+        assert global_contents.get_txt_toolbar_title(set_capabilities).text == values.DISCOVERY_SCREEN_TITLE
+
+        discovery_message = global_contents.get_android_element_by_text(set_capabilities, values.DISCOVERY_SCREEN_MESSAGE)
+        assert discovery_message.text == values.DISCOVERY_SCREEN_MESSAGE
+
+    def test_discovery_search_and_trending(self, set_capabilities, setup_logging):
+        """
+        Scenarios:
+            Verify search field is displayed
+            Verify search field hint text
+            Verify search button is displayed
+            Verify search button text
+            Verify trending label is displayed
+            Verify trending label text
+            Verify python course is displayed
+            Verify python course text
+            Verify excel course is displayed
+            Verify excel course text
+            Verify data course is displayed
+            Verify data course text
+            Verify marketing course is displayed
+            Verify marketing course text
+        """
+
+        global_contents = Globals(setup_logging)
+
+        search_field = global_contents.wait_and_get_element(set_capabilities, android_elements.discovery_search_field)
+        assert search_field.get_attribute('hint') == values.DISCOVERY_SEARCH_FIELD
+
+        search_button = global_contents.wait_and_get_element(set_capabilities, android_elements.discovery_search_button)
+        assert search_button.text == values.DISCOVERY_SEARCH_BUTTON
+
+        trending_label = global_contents.get_element_by_exact_text_android(set_capabilities, values.DISCOVERY_TRENDING_LABEL)
+        assert trending_label.text == values.DISCOVERY_TRENDING_LABEL
+
+        python_label = global_contents.get_element_by_exact_text_android(set_capabilities, values.DISCOVERY_TRENDING_COURSE_PYTHON)
```

Comment:
We should integrate pylint or pycodestyle on this repo for following Pep8 rules.

Reply:
@jawad-khan review is well taken, but integrating pycodestyle or pylint will create a lot of changes in multiple files,
I will do this in a separate PR.